transformer_engine/pytorch/distributed.py

```diff
@@ -812,7 +812,9 @@ def set_states(self, states: Dict[str, torch.Tensor]) -> None:
         Set the rng states. For efficiency purposes, we do not
         check the size of seed for compatibility.
 
-        states: Dict[str, torch.Tensor]
+        Parameters
+        ----------
+        states : Dict[str, torch.Tensor]
```

Comment:
There are other cases where there is no space between the parameter and the ':' - we should use one consistently (preferably the right one :-) ).